packages/cli/src/gemini.tsx

```diff
@@ -222,6 +224,10 @@ export async function main() {
       { exitOnCtrlC: false },
     );
 
+    checkForUpdates().then((info) => {
```

Comment:
You'll want to `.catch` this to ensure we don't have an unobserved exception in the worst case